packages/backend/src/submission/mongoRepository.ts

```diff
@@ -64,20 +65,26 @@ async function addSubmission(collection: Collection, submission: Submission) {
   return submission
 }
 
-async function reviewSubmission(
+async function updateSubmission(
   collection: Collection,
-  id: string,
-  state: 'approved' | 'rejected',
+  submission: Submission,
+  config: { session?: ClientSession; newDatestamps?: boolean } = {
+    newDatestamps: true,
+  },
 ) {
-  const submission = await collection.findOneAndUpdate(
+  const { id, ...data } = submission
+  if (config.newDatestamps) {
```

Comment:
What's the purpose of `newDatestamps`? Shouldn't we update the date every time we do the update? 🤔

Reply:
`newDatestamps` was a property only relevant to the manual revert transaction approach, i.e. to set the `updatedAt` back to what it was before the failed update transaction.

Accidentally copied over to this branch, will remove.